runtime/drivers/olap.go

```diff
@@ -596,7 +596,7 @@ func (d Dialect) GetValExpr(val any, typ runtimev1.Type_Code) (bool, string, err
 	}
 	switch typ {
 	case runtimev1.Type_CODE_STRING:
-		return true, fmt.Sprintf("'%v'", val), nil
+		return true, fmt.Sprintf("%s", d.EscapeStringValue(val.(string))), nil
```

Comment:
Can you make the cast safe? Usually `Type_CODE_STRING` should be a Go `string`, but there may be unpredictable edge cases (maybe around `null`s)

Reply:
null is already checked before switch but will add check anyways

Reply:
added